FantasyCritic.Web/ClientApp/components/pages/masterGames.vue

```diff
@@ -71,6 +71,7 @@
           </template>
         </b-table>
       </div>
+      <Loader :loading="!showGames" />
```

Comment:
Right here is where we could use font-awesome.

<font-awesome-icon icon="circle-notch" size="4x" spin />

Reply:
Or just use use the spinner icon already defined, your call. 

`<font-awesome-icon icon="spinner"  size="4x" pulse  v-if="!showGames" />`

Reply:
That sounds good to me. Update the PR and I will merge it.